### PR TITLE
Add Clover Finance to SS58 Registry

### PR DESCRIPTION
**Pull request by pangwa**
This adds clover ss58(128) address prefix.

### CODE REVIEW

**Comment by pangwa** [comment]
CLA signed

**Comment by pangwa** [comment]
@BenWhiteJam could you please help review it? Thanks!

**Comment by pangwa** [comment]
@BenWhiteJam  @bkchr Hello, it looks like this PR was stucked for some reason. Anything that I do to help get it approved/merged?  Thanks!

**Review by BenWhiteJam — APPROVED** [review]
fine from BD side.

- [x] contact
- [x] testnet
- [x] code repo
- [x] support
- [x] community

File: primitives/core/src/crypto.rs [hunk 1]
```diff
@@ -587,10 +587,12 @@ ss58_address_format!(
 		(98, "polkasmith", "PolkaSmith Canary Network, standard account (*25519).")
 	PolkaFoundry =>
 		(99, "polkafoundry", "PolkaFoundry Network, standard account (*25519).")
-  OriginTrailAccount =>
+    OriginTrailAccount =>
```
Comment (shawntabrizi) [inline primitives/core/src/crypto.rs:590]:
```suggestion
  OriginTrailAccount =>
```

**Comment by shawntabrizi** [comment]
needs merge and space fixes

**Review by bkchr — APPROVED** [review]

**Comment by bkchr** [comment]
bot merge

**Pull request merged**